src/EditorFeatures/Core.Wpf/InlineRename/UI/Adornment/RenameFlyoutViewModel.cs

```diff
@@ -223,7 +224,7 @@ public bool Submit()
         public void Cancel()
         {
             SmartRenameViewModel?.Cancel();
-            Session.Cancel();
+            _ = Session.CancelAsync();
```

Comment:
i'm not a fan of this being fire-forget here.  i think we shoudl make the containing call async, and decide waht to do (on a case by case basis) for every caller.  Some will be able to await, some can be explicit fire-and-forget, and some can block.

Note: Fire-and-forget here is not good for rename tracking as we have untracked async work being kicked off.  anywhere we do this, we need IAsyncOpListener hooked up to keep track of things.